schedulerTools.sh

```diff
@@ -127,4 +127,48 @@ function launch_job {
   rm ${sacct_log}
 }
 
-
+# Function to launch a job with the scheduler, or just run it if the scheduler is nonw
+function run_script {
+    local RUN_CMD=$1
+    local NAME=$2
+    local SCHEDULER_SCRIPT=$3
+    local envdir=`dirname $0`
+    if [ "${scheduler}" = "slurm" ] ; then
```

Comment:
I think most of the code is pretty generic here and if we would pass ${scheduler} to `launch_job` there would be nothing `scheduler`-specific in the code below. So I think we could live with a `if [ "${scheulder}" != "none" ] ; then` here.

Reply:
sounds good